pandas/core/frame.py

```diff
@@ -22,7 +20,24 @@
 import numpy as np
 import numpy.ma as ma
 
+import pandas.core.algorithms as algorithms
+import pandas.core.common as com
+import pandas.core.indexes.base as ibase
+import pandas.core.nanops as nanops
+import pandas.core.ops as ops
+import pandas.io.formats.console as console
+import pandas.io.formats.format as fmt
+import pandas.plotting._core as gfx
+from pandas import compat
+from pandas._libs import lib, algos as libalgos
+from pandas.compat import PY36
+from pandas.compat import (range, map, zip, lrange, lmap, lzip, StringIO, u,
```

Comment:
Combine with previous

Reply:
Thanks done!

Reply:
This is great, thanks, I know its nitpicky.  For future reference in case you want to _really_ match my habits, I usually put a newline after the compat imports, then collect core.dtypes imports in a section above the rest of core (lots of stuff depends on core.dtypes, but it depends on very little except for compat and _libs)

Reply:
Thanks much appreciated, I will update! I might add some kind of description to the contributing guide on the import layout you have described here, unless its already documented somewhere?